sesame/src/main/scala/org/w3/banana/sesame/SesameOps.scala

```diff
@@ -167,7 +167,7 @@ class SesameOps extends RDFOps[Sesame] with SesameMGraphOps with DefaultURIOps[S
   def isomorphism(left: Sesame#Graph, right: Sesame#Graph): Boolean = {
     val leftNoContext = left.asScala.map(s => makeTriple(s.getSubject, s.getPredicate, s.getObject)).asJava
     val rightNoContext = right.asScala.map(s => makeTriple(s.getSubject, s.getPredicate, s.getObject)).asJava
-    ModelUtil.equals(leftNoContext, rightNoContext)
+    Models.isomorphic(leftNoContext, rightNoContext)
```

Comment:
The comment above is no longer accurate since `ModelUtil` is no longer used. So the comment needs to be adapted or removed.

```
 /**
   * the new ModelUtil.equals changed its semantics. See
   * - https://openrdf.atlassian.net/browse/SES-1695
   * - https://groups.google.com/forum/#!topic/sesame-devel/CGFDn7mESLg/discussion
   */
```